app/src/main/java/com/hamdan/forzenbook/data/network/CreateAccountService.kt

```diff
@@ -4,13 +4,13 @@ import retrofit2.Response
 import retrofit2.http.POST
 
 interface CreateAccountService {
-    @POST("/createUser")    //ToDo(need to add the email to this path, ask nic what the post should look like for this)
+    @POST("/createUser") // ToDo(need to add the email to this path, ask nic what the post should look like for this)
```

Comment:
Constant for endpoint

Reply:
fair

Reply:
oh wait thats for the CreateAccount, thats not in this ticket. I made it anyway but it is out of scope.